tests/calico_testcases/example_policies/testcase24-selector-expression/testcase24-scheme.yaml

```
@@ -0,0 +1,29 @@
+namespaceList: ../../example_podlist/ns_list_wdc.json
+podList: ../../example_podlist/hep_and_wep.json
+
+networkConfigList:
+  - name: np_a
+    networkPolicyList:
+      - testcase24-a.yaml
+      - testcase24-profiles.yaml
+    expected_warnings: 0
```

```suggestion
    expectedWarnings: 0
```
Shireen changed the keyword here. Make sure you merge with `master`